nodebuilder/state/state.go

```diff
@@ -215,52 +197,46 @@ func (api *API) SubmitTx(ctx context.Context, tx state.Tx) (*state.TxResponse, e
 
 func (api *API) SubmitPayForBlob(
 	ctx context.Context,
-	fee state.Int,
-	gasLim uint64,
-	blobs []*blob.Blob,
+	blobs []*state.Blob,
```

Comment:
Specifying a list of things we break in this PR would be nice. This one is minor but very breaking. Some teams rely on this endpoint isntead of blob.Submit

Reply:
I will modify PR description